LICENSE.md

```diff
@@ -78,7 +78,7 @@ _This file was generated by the `bin/docs.py` script on 2023-07-26._
   - **India**: German Aerospace Center (DLR)
 - **City of Kuopio**: source [data](https://www.avoindata.fi/data/fi/dataset/kuopion-kaupunginosat/resource/6ca89290-3743-4832-9ed6-03d8cf9b2d5f) and [license](https://www.avoindata.fi/data/fi/dataset/kuopion-kaupunginosat)
 - **City of Los Angeles Neighborhood Councils (Certified)**: source [data](https://data.lacity.org/A-Well-Run-City/Neighborhood-Councils-Certified-/fu65-dz2f) and [license](https://data.lacity.org/A-Well-Run-City/Neighborhood-Councils-Certified-/fu65-dz2f/about)
-- **Luxembourg: Administration du cadastre et de la topographie**: source [data](https://data.public.lu/en/datasets/limites-administratives-du-grand-duche-de-luxembourg/#) and [license](https://data.public.lu/)
+- **Luxembourg: Administration du cadastre et de la topographie**: source [data](https://data.public.lu/en/datasets/limites-administratives-du-grand-duche-de-luxembourg/#) and [license](https://data.public.lu/en/datasets/limites-administratives-du-grand-duche-de-luxembourg/#)
```

Comment:
Nit: The trailing `/#` in the two links aren't neccesary.

Reply:
Good catch - removing those in a new commit.